dnn_test.go

```diff
@@ -47,6 +47,11 @@ func TestReadNet(t *testing.T) {
 			t.Errorf("Invalid len output layers in ReadNet test: %d\n", len(ids))
 		}
 
+		lnames := net.GetLayerNames()
+		if len(lnames) != 1 {
```

Comment:
For the model being tested here, the number of layers is 142.

I think if this number is changed in the test, the CI will pass and this PR could be merged...